taskcluster/client.py

```diff
@@ -603,5 +295,12 @@ def createTemporaryCredentials(clientId, accessToken, start, expiry, scopes, nam
 __all__ = ['createTemporaryCredentials', 'config']
 # This has to be done after the Client class is declared
 for key, value in API_CONFIG.items():
-    globals()[key] = createApiClient(key, value)
+    # import buildtime generated code if it exists; otherwise use runtime generated code
+    try:
+        module = __import__('taskcluster.sync.{key}'.format(key=key), globals(),
+                            locals(), [key], 0)
+        globals()[key] = getattr(module, key)
+    except (ImportError, AttributeError):
+        print("{} not in taskcluster.sync!!!".format(key))
```

Comment:
+1, done.  `test_generated.py` already tests for missing classes, so this is unneeded.